docker/osd-dev/dev.sh

```diff
@@ -1,42 +1,5 @@
 #!/bin/bash
 
-os_versions=(
-  '1.2.4'
-  # '2.0.0' # compatibility.override_main_response_version deprecated
-  '2.1.0'
-  '2.2.1'
-  '2.3.0'
-  '2.4.0'
-  '2.4.1'
-  '2.6.0'
-  '2.8.0'
-  '2.9.0'
-  '2.10.0'
-  '2.11.0'
-  '2.11.1'
-  '2.12.0'
-  '2.13.0'
-
-)
-
-osd_versions=(
-  '1.2.0'
-  # '2.0.0' # compatibility.override_main_response_version deprecated
-  '2.1.0'
-  '2.2.1'
-  '2.3.0'
-  '2.4.0'
-  '2.4.1'
-  '2.6.0'
-  '2.8.0'
-  '2.9.0'
-  '2.10.0'
-  '2.11.0'
-  '2.11.1'
-  '2.12.0'
-  '2.13.0'
-)
-
 wzs_version=(
```

Comment:
I think I would take this out so that I don't have to update every time a new version of wazuh comes out. just as you did with the opensearch releases